### PR TITLE
add warning to FIPS chapter that MD5 is unsupported for Samba/CIFS

### PR DESCRIPTION
**Pull request by cjschroder**
### PR creator: Description
Add warning to FIPS chapter that MD5 is unsupported for Samba/CIFS. 

### PR creator: Are there any relevant issues/feature requests?

jira-SLE-23133

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE 15/openSUSE Leap 15.x
  - [x] SLE 15 SP4/openSUSE Leap 15.4 *(current `main`, no backport necessary)*
  - [x] SLE 15 SP3/openSUSE Leap 15.3
  - [x] SLE 15 SP2/openSUSE Leap 15.2
  - [x] SLE 15 SP1
  - [x] SLE 15 GA
- SLE 12
  - [x] SLE 12 SP5
  - [x] SLE 12 SP4
  - [x] SLE 12 SP3

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

### CODE REVIEW

**Comment by cjschroder** [comment]
Here is a PDF for easier reviewing:
[Uploading cha-security-fips_color_draft_en.pdf…]()

**Review by denknorr — CHANGES_REQUESTED** [review]

File: xml/security_fips.xml [hunk 1]
```diff
@@ -175,13 +175,29 @@ crypto.fips_enabled = 0</screen>
...
  </sect1>
-</chapter>  
\ No newline at end of file
+
+ <sect1 xml:id="sec-fips-samba-cifs">
+  <title>MD5 not supported in Samba/CIFS</title>
+  <para>
+   According to the FIPS standards, MD5 is not a secure hashing
+   algorithm, and it must not be used for authentication. If you run a
+   FIPS-compliant network environment, and your clients and servers run in
```
[the remark this answers is not in the document]
Comment (cjschroder) [reply xml/security_fips.xml:193]: good point, I will make it 'and you have clients or servers...'

**Review by denknorr — APPROVED** [review]

**Comment by cjschroder** [comment]
SLE15SP3 37600dd60
SLE15SP2 1d6c35729
SLE15SP1 37431138d
SLE15SP0 61981eff2

**Comment by cjschroder** [comment]
related: https://github.com/SUSE/doc-sle/pull/1181/

**Pull request merged**